src/icon-button.jsx

```diff
@@ -54,28 +54,83 @@ const IconButton = React.createClass({
   },
 
   propTypes: {
+    /**
+     * Can be used to pass a font icon as the icon for the button.
+     */
     children: React.PropTypes.node,
 
     /**
      * The css class name of the root element.
      */
     className: React.PropTypes.string,
+
+    /**
+     * Disables the icon button.
+     */
     disabled: React.PropTypes.bool,
+
+    /**
+     * If you are using a stylesheet for your
+     * icons, enter the class name for the icon to be used here.
+     */
     iconClassName: React.PropTypes.string,
+
+    /**
+     * Overrides the inline-styles of the icon element.
+     */
     iconStyle: React.PropTypes.object,
+
+    /**
+     * Callback function for when the component loses focus.
+     */
     onBlur: React.PropTypes.func,
+
+    /**
+     * Callback function for when the component gains focus.
+     */
     onFocus: React.PropTypes.func,
+
+    /**
+     * Callback function for when the component
+     * receives keyboard focus.
+     */
     onKeyboardFocus: React.PropTypes.func,
+
+    /**
+     * Callback function for when mouse enters element.
+     */
     onMouseEnter: React.PropTypes.func,
+
+    /**
+     * Callback function for when mouse leaves element.
+     */
     onMouseLeave: React.PropTypes.func,
 
     /**
      * Override the inline-styles of the root element.
      */
     style: React.PropTypes.object,
+
+    /**
+     * The tooltip text to show.
+     */
     tooltip: React.PropTypes.node,
+
+    /**
+     * Allows the tooltip to be viewed with different
+     * alignments: "bottom-center", "top-center",
+     * "bottom-right", "top-right", "bottom-left" and "top-left".
+     */
     tooltipPosition: PropTypes.cornersAndCenter,
+
+    /**
+     * Styles prop passed down to the tooltip.
+     */
     tooltipStyles: React.PropTypes.object,
+
+    /**
+     * Prop for tooltip to make it larger for mobile.
+     */
     touch: React.PropTypes.bool,
```

Comment:
yes, @Zadielerick All boolean props must have a default. Makes them easier to understand :grin: